frontend/lib/src/components/elements/ArrowVegaLiteChart/arrowUtils.ts

```diff
@@ -84,10 +84,8 @@ export interface WrappedNamedDataset {
 }
 
 export function getInlineData(
-  el: VegaLiteChartElement
+  dataProto: Quiver | null
```

Comment:
nit: it was named `dataProto` before, but I think the name is a bit misleading since it doesn't have a lot to do with protobuf anymore at that point. The Quiver is a processed form of the arrow data which comes from a proto message, but there isn't any pointer anymore to the protobuf message itself. But I don't have a strong opinion to keep or not to keep the prefix.